src/Operation/Reject.php

```diff
@@ -80,11 +55,7 @@ static function (Iterator $iterator) use ($callbacks): Generator {
                         $callbacks;
 
                     foreach ($iterator as $key => $current) {
-                        $result = array_reduce(
-                            $callbacks,
-                            $reducerCallback($key)($current)($iterator),
-                            false
-                        );
+                        $result = OrCallbackReducer::or()($callbacks, $current, $key, $iterator);
```

Comment:
I think having `Or` twice in this is a bit redundant.

I propose either:

```suggestion
                        $result = CallbackReducer::or()($callbacks, $current, $key, $iterator);
```

this leaves room to have `CallbackReducer::and()`

OR 

```suggestion
                        $result = OrCallbackReducer::of()($callbacks, $current, $key, $iterator);
```

and we make it invokable

Reply:
Fair enough, done !